README.md

```diff
@@ -43,59 +44,94 @@ Contains the 3D models of the external components and their supports
   - **D435i_case.sldprt**: Realsense D435i protective case
   - **Intel_RealSense_Depth_Camera_D435.SLDASM**: 3D model of the camera distributed by Realsense [CAD Files (intelrealsense.com)](https://dev.intelrealsense.com/docs/cad-files)
   - **D435i_support.sldprt**: Camera stand that fits on a bolt pattern of the pioneer's top plate. The protective case is compatible with the support.
+  - **D435i.SLDASM**: Assembly of the camera, its protective case and its support.
   - **protected_D435i.SLDASM**: Assembly of the camera and its protective case.
+  
 - **D455**
+  
   - **D455_case.sldprt**: Realsense D455 protective case
   - **Intel_RealSense_D455_CAD_external_v2.SLDASM**: 3D model of the camera distributed by Realsense [CAD Files (intelrealsense.com)](https://dev.intelrealsense.com/docs/cad-files)
   - **D455_support.sldprt**: Camera stand that fits on a bolt pattern of the pioneer's top plate. The protective case is compatible with the support.
+  - **D455_assembly.SLDASM**: Assembly of the camera, its protective  case and its support.
   - **protected_D455.SLDASM**: Assembly of the camera and its protective case.
+  
 - **motor_stop.SLDPRT**
+  
   - Cad of the pushbutton used to stop the motor power. Is mounted in a 22mm hole in Pioneer's top plate.
+  
 - **Lidar**
   - **rplidar_simplified.SLDPRT**: 3D model of a simplified Rplidar A2M8.
-  - **lidar_support.SLDPRT**: Lidar stand. Our Pioneer 2Dx has already a bolt pattern on its top plate that we reused to save us time.
+  - **lidar_support_april_tags.SLDPRT**: Lidar stand. Our Pioneer 2Dx has already a bolt pattern on its top plate that we reused to save us time. Its sides were designed to install a plate or a sheet with AprilTtags on them.
```

Comment:
We reused the already present bolt partern on the top plate of our Pionneer 2Dx to save time. The sides were designed to install a plate or sheet with AprilTags on them.